### PR TITLE
update!: Health and fitness guide

### LINKED ISSUE #1562: Add "Personal Health" section
We should look at a section covering personal health/fitness tracking and management, including period tracking as mentioned below, as well as just a generalized health database. Outside of fitness tracking, health databases have a clear use-case for a variety of medical reasons such as medication, vitals, and nutrition tracking; so this definitely isn't something that can be dismissed as simply as "just don't track that stuff if you want privacy" lol

On iOS, simply sticking with Apple Health seems like a clear recommendation, as it operates entirely on-device and includes period tracking as well as tracking of essentially any other health metric. I think we need to research available options on Android still, including the ones linked below:

### Discussed in https://github.com/orgs/privacyguides/discussions/1513

<div type='discussions-op-text'>

<sup>Originally posted by **FriskyInitiate** July  5, 2022</sup>
Period tracker apps are used to record highly personal information that can include your menstruation cycle, sexual activity, daily mood, pain levels, whether you are trying to have a baby, pregnancy or if you experience a miscarriage. With the recent attack on reproductive rights in the US, it seems more relevant than ever to introduce a category for period tracking apps. Consumer Reports has [evaluated](https://web.archive.org/web/20220630092722/https://www.consumerreports.org/health-privacy/what-your-period-tracker-app-knows-about-you-a8701683935/) some of the available apps for privacy invasions and [recommends](https://web.archive.org/web/20220630215118/https://www.consumerreports.org/health-privacy/period-tracker-apps-privacy-a2278134145/) [Drip](https://bloodyhealth.gitlab.io/), [Euki](https://eukiapp.com/) and [Periodical](https://arnowelzel.de/en/projects/periodical).</div>

### PR DESCRIPTION
**Pull request by jonaharagon**
Closes #1562

### CODE REVIEW

**Comment by d4rklynk** [comment]
You can add the fact that Apple health has [e2ee support](https://support.apple.com/en-us/HT202303)

**Comment by TrashPandaCodingGarbage** [comment]
Why is Sleep as Android recommended? Privacy policy of the service looks not so good on the first glance. I recommend replacing it with [plees](https://f-droid.org/en/packages/hu.vmiklos.plees_tracker/).

**Comment by KaiTebay** [comment]
> Why is Sleep as Android recommended? Privacy policy of the service looks not so good on the first glance. I recommend replacing it with [plees](https://f-droid.org/en/packages/hu.vmiklos.plees_tracker/).

**Tl;dr: I think the privacy policy seems reasonable for what I expect the majority of user's threat models to be. I've summarized as best I can, and my main notables are opt-out defaults for anonymous usage analytics and personalized ads. I haven't tested it, but I suspect that disabling network access would be also be effective.**

I'm not an expert but Urbandroid's privacy policy seems to commit them to:
- User-identifying information for bug reports; user-generated app data for backups; anonymous usage analytics; coarse geo location data (+/- 10km), stored locally (except for inclusion with user-approved backup via SleepCloud).
- Ability to delete user data (varies for third-parties). Usage analytics cannot be deleted (although they can be turned off) because Urbandroid claims that they are anonymous data.
- No transfer of personally identifiable information to third parties, except those necessary for services (i.e. the ones listed) or for legal compliance.

- **Third-parties:**

**Opt-out:**

1. _Facebook Audience Network_, via the Device Advertising ID.
2. _Google Analytics for Firebase_, for anonymous usage analytics.

---

**Opt-in:**

1. _Gmail_ and _Help Scout_ for bug reports.
2. Data backup via _Google Drive_.
3. Data backup via _Dropbox_.
4. Data backup via _SleepCloud_, owned and maintained by Urbandroid; uses _Google Cloud Platform_ (Datastore).
5. _Google Fit_.

### Summary:
The majority of data usage seems to be opt-in integrations with third-party services. Two usages stood out to me: personalized ads, and anonymous usage analytics. It sounds like they are enabled by default but it seems that they can be disabled in the app's settings. The coarse geo location data seems to be stored locally, unless the user approves backup via SleepCloud.

File: docs/health-and-fitness.en.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+### Drip
+
+### Euki
+
+### Periodical
+
+## Sleep Tracking
+
+Apple Watch users should consider using WatchOS' native sleep tracking functionality. Using native, locally-based solutions instead of third-party apps reduces your attack surface by limiting the number of parties that may have access to or be responsible for protecting your data.
```
Comment (purtual) [inline docs/health-and-fitness.en.md:60]:
```suggestion
Apple Watch users should consider using watchOS' native sleep tracking functionality. Using native, locally-based solutions instead of third-party apps reduces your attack surface by limiting the number of parties that may have access to or be responsible for protecting your data.
```

**Review by purtual — CHANGES_REQUESTED** [review]

**Comment by jonaharagon** [comment]
a mention I don't want to forget https://www.reddit.com/r/PrivacyGuides/comments/11lhyzl/comment/jd4kgap/?utm_source=share&utm_medium=web2x&context=3

**Comment by freddy-m** [comment]
@privacyguides/team what is the status of this PR? Are we still interested in adding this page or...

**Comment by dngray** [comment]
> what is the status of this PR? Are we still interested in adding this page or...

Yes we're going to add it, though it has not been reviewed yet. Working through the pages in queue.

**Comment by friadev** [comment]
> @friadev @redoomed1 you mind also having a look at this one. I do really hope to have this added one day.

Yeah I’d love to, I don’t have access to Android at the moment.
